app/src/main/kotlin/com/wire/android/ui/home/newconversation/NewConversationViewModel.kt

```diff
@@ -193,7 +195,10 @@ class NewConversationViewModel @Inject constructor(
                 options = ConversationOptions().copy(
                     protocol = newGroupState.groupProtocol,
                     readReceiptsEnabled = groupOptionsState.isReadReceiptEnabled,
-                    accessRole = groupOptionsState.accessRoleState
+                    accessRole = groupOptionsState.accessRoleState,
+                    access = if (groupOptionsState.accessRoleState.contains(Conversation.AccessRole.GUEST)) {
+                        Conversation.defaultGroupAccess.toMutableSet().apply { add(Conversation.Access.CODE) }
+                    } else null
```

Comment:
Wasn't this already included on this previous [PR](https://github.com/wireapp/wire-android-reloaded/pull/1536)?

Reply:
Yes, just nicer. Uses a convenience function